ui/packages/consul-ui/mock-api/v1/acl/binding-rules

```diff
@@ -0,0 +1,30 @@
+[
+  ${
+    range(
+      env(
+        'CONSUL_BINDING_RULE_COUNT',
+        Math.floor(
+          (
+            Math.random() * env('CONSUL_BINDING_RULE_MAX', 10)
+          ) + parseInt(env('CONSUL_BINDING_RULE_MIN', 1))
+        )
+      )
+    ).map(
+      function(item, i) {
+        return `
+          {
+            "ID": "${fake.random.uuid()}",
+            "Description": "${fake.lorem.sentence()}",
+            "AuthMethod": "${fake.hacker.noun()}-${i}",
+            "Selector": "serviceaccount.namespace==default and serviceaccount.name!=vault",
+            "BindType": "${fake.helpers.randomize(['service', 'node', 'role'])}",
+            "BindName": "${fake.hacker.noun()}-${i}",
```

Comment:
I read in the API docs that these can contain HIL, should we randomly make some of these contain HIL?

Reply:
It can be either. I did the easier version.